napari/layers/utils/_tests/test_text_manager.py

```diff
@@ -121,7 +121,9 @@ def test_text_manager_format():
 
     text_manager.anchor = 'center'
     coords = np.array([[0, 0], [10, 10], [20, 20]])
-    text_coords = text_manager.compute_text_coords(coords, ndisplay=3)
+    text_coords = text_manager.compute_text_coords(
+        coords, ndisplay=3, order=tuple(range(3))
+    )
```

Comment:
Would be great to add an entirely separate test case to cover the ordering of the text coordinates (i.e. just targeting `compute_text_coords`). I can try to find some time to add one if you don't get to it.